build-tools/packages/build-cli/src/flags.ts

```diff
@@ -50,6 +51,18 @@ export const releaseGroupFlag = Flags.custom<ReleaseGroup>({
 	},
 });
 
+/**
+ * A re-usable CLI flag to parse release group name.
+ */
+export const releaseGroupNameFlag = Flags.custom<ReleaseGroupName>({
+	name: "release_group",
```

Comment:
You can omit this - since it's a flag the name is inferred from the key where the flag is used.